lib/client-assets.php

```diff
@@ -182,36 +182,6 @@ function gutenberg_override_style( $styles, $handle, $src, $deps = array(), $ver
 	$styles->add( $handle, $src, $deps, $ver, $media );
 }
 
-/**
- * Registers vendor JavaScript files to be used as dependencies of the editor
- * and plugins.
- *
- * This function is called from a script during the plugin build process, so it
- * should not call any WordPress PHP functions.
- *
- * @since 0.1.0
- *
- * @param WP_Scripts $scripts WP_Scripts instance.
- */
-function gutenberg_register_vendor_scripts( $scripts ) {
```

Comment:
With React 18, we might need to move that to another folder, but it's a problem for later :)